### ISSUE TITLE
Render points as spheres

### ISSUE THREAD

**Issue by banesullivan** [InsightSoftwareConsortium/itk-jupyter-widgets#180]
Is it possible to render points as spheres? Here's a good example dataset:

```py
from pyvista import examples
from itkwidgets import view

mesh = examples.download_lidar()
view(geometries=mesh)
```

<img width="928" alt="Screen Shot 2019-07-26 at 8 17 35 AM" src="https://user-images.githubusercontent.com/22067021/61958068-dcb2f800-af7d-11e9-9164-5e6ae5a0dfcf.png">

if not, this is a feature request

**Comment by thewtex** [comment]
Yes, with #188, we can theoretically use spheres by passing `point_sets=` as opposed to `geometries=`:

```
view(point_sets=mesh)
```

This provides a UI to change the point size, and an option to view as spheres.

However, when viewing as sphere glyphs, that dataset crashes the browser. So, https://github.com/Kitware/itk-vtk-viewer/issues/164 will need to be addressed in practice.
